.travis.yml

```diff
@@ -1,7 +1,12 @@
 language: java
+
+sudo: false
+
 jdk:
-  - oraclejdk8
-  - openjdk8
+- openjdk8
+- openjdk10
+- oraclejdk8
+- oraclejdk10
```

Comment:
Seems like a lot of maybe unnecessary overlap for all those tests

Reply:
could perhaps drop oraclejdk then?